src/ui-test/tests/lsp_extension.test.ts

```diff
@@ -56,7 +56,13 @@ describe('Language Support for Apache Camel extension', function () {
 		});
 
 		it('Find extension', async function () {
-			item = await marketplace.findExtension(`@installed ${pjson.displayName}`);
+			this.retries(5);
+			try {
+				item = await marketplace.findExtension(`@installed ${pjson.displayName}`);
```

Comment:
I guess the findExtension method can be improved directly in ExtensionTester

Reply:
actually MarketPlace extension comes from vscode-uitests-tooling (https://github.com/djelinek/vscode-uitests-tooling/blob/main/src/components/custom/Marketplace.ts#L68) but anyway it can be improved for sure on side of dependency whatever dependency it is 🙂